test/support/application_fixture_helper.rb

```diff
@@ -0,0 +1,75 @@
+# typed: ignore
+# frozen_string_literal: true
+
+module ApplicationFixtureHelper
+  TEMP_FIXTURE_DIR = ROOT.join("tmp", "fixtures").to_s
```

Comment:
I'm writing the temporary data in packwerk's own /tmp directory, instead of the system's temporary directory. While I was debugging tests, I found it quite convenient to see the temporary files right in the project, using my code editor. It's not so fun to figure out where the fixtures go in the system temp directory.

Also, if ever there is a leak of sorts with temporary fixture directories, it's easier to spot, being right in the project workspace.

Reply:
I that case can you add it to the .gitignore file so we don't commit it accidentally?

Reply:
If `/tmp/` is already on that list, I believe that's already covered.